pkg/mimirtool/commands/alerts_test.go

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/grafana/cortex-tools/blob/main/pkg/commands/alerts_test.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Cortex Authors.
```

Comment:
Is this necessary? the code in this file seems authored by you and that cortex link doesn't eixst.

Reply:
I copied it from a different file. All the files in this directory had this, so I was unsure, and copied it over just in case.

Will remove.